docs/docs/rest-api/public/api/v2/types/pragma.raml

```diff
@@ -39,5 +39,8 @@ annotationTypes:
       Mark a type as internal to indicate, that it is not used in the external API.
       No functional change - only to indicate the use of one type.
       Use only on type level.
-
+  serializeOnly:
+    type: nil
```

Comment:
why `nil`?

Reply:
b/c the flag will be added w/o any value. That's how other similar flags are defined and used.